.env.sample

```diff
@@ -9,6 +11,7 @@ ES_ENGINE=
 
 # For elasticsearch cluster
 CLOUD_ID=
+API_KEY=
```

Comment:
@kouloumos No need to add a new variable `API_KEY` as we are using it as `USER_PASSWORD` in the script.

Reply:
Okay, I'll keep one, but is this actually a user password? or it's an api key? it's good to have descriptive names for our variables, especially ENV variables.

Reply:
I agree, it's time we begin updating it, let's start using `API_KEY` whenever we edit the script. For now we will also keep the `USER_PASSWORD` so that I does not break the workflow.